src/index.js

```diff
@@ -28,14 +28,24 @@ import 'styles/main.scss';
       const $selection = $this.find('p.iqdropdown-selection').last();
       const $menu = $this.find('div.iqdropdown-menu');
       const $items = $menu.find('div.iqdropdown-menu-option');
-      const settings = $.extend(true, {}, defaults, options);
       const itemCount = {};
       let totalItems = 0;
 
+      const settings = $.extend(
+        true,
+        {
+          getCustomMessage: () => {
```

Comment:
I think it would be better if this function was declared in the default object along with the other defaults. this way the function parameters would be clearer

Reply:
Move function to default object. Use `this` to get `selectionText` and `textPlural` values.